docs/dk-api.postman_environment.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Wipe this commit as the actual server url is inside. The development url and token can vary from person to person so no point committing this.

Reply:
ok